libs/execution_context/ebpf_native.c

```diff
@@ -121,7 +121,7 @@ static const NPI_PROVIDER_CHARACTERISTICS _ebpf_native_provider_characteristics
     _ebpf_native_provider_detach_client_callback,
     NULL,
     {
-        EBPF_PROGRAM_INFORMATION_PROVIDER_DATA_VERSION,
+        0,
```

Comment:
Ok but in other places you use a *_VERSION_0 define.  For consistency, why not here?

Reply:
Because [MSDN](https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/netioddk/ns-netioddk-_npi_provider_characteristics) says:
`The version of the NMR with which the provider is registering. A provider module should set this member to zero.`